### PR TITLE
Add New Domain threatbook.cn

### PR DESCRIPTION
**Pull request by sinlee1**

### CODE REVIEW

File: data/threatbook
Comment (IceCodeNew) [inline data/threatbook:3]: The last two domains are covered by the first domain, please delete those two subdomains.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

**Comment by IceCodeNew** [comment]
BTW, I can't see why we make a category for JUST one domain, especially when that domain has a `.cn` TLD.

**Comment by database64128** [comment]
> BTW, I can't see why we make a category for JUST one domain, especially when that domain has a `.cn` TLD.

I don't use this particular domain, but being able to use `geosite:xxx` instead of having to spell out the full domain can be quite convenient.

**Comment by IceCodeNew** [comment]
> > BTW, I can't see why we make a category for JUST one domain, especially when that domain has a `.cn` TLD.
> 
> I don't use this particular domain, but being able to use `geosite:xxx` instead of having to spell out the full domain can be quite convenient.

That's exactly what I am asking. If you are trying to whitelist this domain, [geolocation-cn](https://github.com/v2fly/domain-list-community/blob/master/data/cn) should have covered it.
If you are including this domain for some different use-case, then I must know that before I can approve this PR.

**Comment by sinlee1** [comment]
Modified

File: data/threatbook
Comment (IceCodeNew) [inline data/threatbook:1]: If you are trying to whitelist this domain, geolocation-cn should have covered it.
If you are including this domain for some different use-case, then I must know that before I can approve this PR.

**Review by IceCodeNew — CHANGES_REQUESTED** [review]

File: data/threatbook
Comment (sinlee1) [reply data/threatbook:1]: I don't see where the geolocation-cn file is located. If I just want to customize the domain access how do I set it up?

Comment (IceCodeNew) [reply data/threatbook:1]: > I just checked geolocation-cn and found that there is no such domain inside, so I want to add it

`tld-cn` will cover all `.cn` domains.

Comment (sinlee1) [reply data/threatbook:1]: Understood, the problem is solved. How do I add a domain name for my personal needs website? If I encounter a domain that does not exist inside geolocation-cn, is it possible to pr submit it again?

Comment (IceCodeNew) [reply data/threatbook:1]: > How do I add a domain name for my personal needs website?

You can generate your own geosite.dat file. https://github.com/Loyalsoldier/v2ray-rules-dat is a good place to start.

---

> If I encounter a domain that does not exist inside `geolocation-cn`, is it possible to pr submit it again?

Sure. Just issue another PR and wait for someone to review it ;-)

**Pull request closed without merging**
